src/mock/model/points-model.js

```diff
@@ -31,6 +34,36 @@ export default class PointModel {
   }
 
   getOffersByType(type) {
-    return this.#offers.find((offer) => offer.type === type).offers;
+    return this.#offers.find((offer) => offer.type === type).offers || '';
```

Comment:
ты не понимаешь, что тебе возвращается массив чего-то. Сделай правильный фолбэк
```suggestion
    return this.#offers.find((offer) => offer.type === type)?.offers || [];
```